Remove UniFi Protect docs for using EA versions (#24857)
home-assistant/home-assistant.github.io: source/_integrations/unifiprotect.markdown

* Any UniFi "Dream" device (**UDMPRO**, **UDR**, or **UDMSE**), _except the base UniFi Dream Machine/UDM_
* UniFi Cloud Key Gen2 Plus (**UCKP**) firmware version v2.0.24+

UCKP with Firmware v1.x **do NOT run UniFi OS**, you must upgrade to firmware `v2.0.24` or newer.

### Software Support

The absolute **minimal** software version is `v1.20.0` for UniFi Protect. If you have an older version, you will get errors trying to set up the integration. However, the general advice is the latest 2 minor versions of UniFi Protect and hardware supported by those are supported.

#### About UniFi Early Access

> Since UniFi Protect has its own release cycle, Early Access versions of UniFi Protect are used to ensure compatibility with Home Assistant. As a result, Early Access releases are not supported by Home Assistant.
> 

Reviewer: ```suggestion

```

Reviewer: `Early Access versions of UniFi Protect are used to ensure compatibility with Home Assistant` is probably something they don't need to know since we aren't expecting them to be QA

Reviewer: I did a suggestion below to try to combine this https://github.com/home-assistant/home-assistant.io/pull/24857#discussion_r1020915433

> <div class='note warning'>
> 
> **Early Access releases are not supported by Home Assistant.** Using an them may cause your UniFi Protect integration to break unexpectedly.
> 
> </div>

Reviewer: ```suggestion
<div class='note warning'>
**Early Access releases are not supported by Home Assistant.** 

Using Early Access versions will likely cause your UniFi Protect integration to break unexpectedly.
</div>
```
